app/terminal/index.html

```diff
@@ -44,6 +44,9 @@
           cursorBlink: true,
           theme: theme
       });
+      if ("%6") {
```

Comment:
Use "" is not empty string

Reply:
like ```(eaf-setq eaf-terminal-font-family "")```?

Reply:
yes

Reply:
![image](https://user-images.githubusercontent.com/2212586/96945903-8b434100-1511-11eb-8a18-34a56e0b9153.png)

After ```(eaf-setq eaf-terminal-font-family "")```, the rendered index.html is like this, and ```if ("")``` is no problem, so what i miss?

Reply:
I see, patch is merged, thanks for PR ;)